test/group-decorate.spt

```diff
@@ -38,6 +35,4 @@
 ; RUN: llvm-spirv -r %t.spv -o %t.bc
 ; RUN: llvm-dis < %t.bc | FileCheck %s --check-prefix=CHECK-LLVM
 
-; CHECK-LLVM: define spir_kernel void @test(<4 x i8> addrspace(1)* nocapture %src1, <4 x i8> addrspace(1)* nocapture %src2, <4 x i8> addrspace(1)* nocapture %dst) {{.*}} !kernel_arg_type_qual ![[QUALS:[0-9]+]]
-
-; CHECK-LLVM: ![[QUALS]] = !{!"const", !"const", !""}
+; CHECK-LLVM: define spir_kernel void @test(<4 x i8> addrspace(1)* nocapture %src1, <4 x i8> addrspace(1)* nocapture %src2, <4 x i8> addrspace(1)* nocapture %dst) {{.*}}
```

Comment:
May we add string "const" to keep constant qualifier in this test?

Reply:
This change relates to https://github.com/KhronosGroup/SPIRV-LLVM-Translator/pull/996/commits/b1d6306a9c4904cf3921c9dbebf2bfc673deb622 or https://github.com/KhronosGroup/SPIRV-LLVM-Translator/pull/989 . The test checks correctness of decoration groups and function parameter attribute NoWrite previously is among decorations to be grouped. As we unmap it from 'constant' qualifier we shan't check it in this tests.